website/docs/externaldata.md

```diff
@@ -146,6 +146,9 @@ If there is a system error, the provider should return the system error message
 
 Example provider implementation: https://github.com/open-policy-agent/gatekeeper/blob/master/test/externaldata/dummy-provider/provider.go
 
+#### Caching
+Gatekeeper now supports caching of responses from external data providers. It caches the response based on the `Key` and `Value` received as part of the [`ProviderResponse`](#providerresponse). By default, the cache is invalidated after 3 minutes, which is the default Time-to-Live (TTL). You can configure the TTL using the --external-data-provider-response-cache-ttl flag.
```

Comment:
is this configurable in helm chart today? if not, can you open an issue please?

Reply:
https://github.com/open-policy-agent/gatekeeper/issues/2977